src/main/java/racingcar/car/Car.java

```diff
@@ -32,4 +32,13 @@ public int getPosition() {
 	public String getName() {
 		return this.name;
 	}
+
+	@Override
+	public int compareTo(Car otherCar) {
+		return Integer.compare(this.position , otherCar.getPosition());
+	}
+
+	public boolean isEqualPosition(int otherPosition){
+		return this.position == otherPosition ? true : false;
+	}
```

Comment:
프로그래밍 요구사항으로 **else 예약어를 쓰지 않는다.** 는 조건이 존재합니다.
- 힌트: if 조건절에서 값을 return하는 방식으로 구현하면 else를 사용하지 않아도 된다.
- else를 쓰지 말라고 하니 switch/case로 구현하는 경우가 있는데 switch/case도 허용하지 않는다.

삼항연산자 또한 if-else와 같기 때문에 다음 미션부터 다음과 같이 return 하는 방식으로 진행하시면 좋을 것 같아요!
 ```suggestion
	public boolean isEqualPosition(int otherPosition){
	        if (this.position == otherPosition) {
	            return true;
	        }
	        return false;
	}
```